src/main/java/games/strategy/engine/chat/ChatMessagePanel.java

```diff
@@ -251,7 +249,7 @@ public void addMessageWithSound(final String message, final String from, final b
 
   private void addChatMessage(final String originalMessage, final String from, final boolean thirdperson) {
     final String message = trimMessage(originalMessage);
-    final String time = simpleDateFormat.format(new Date());
+    final String time = "(" + TimeManager.getLocalizedTime() + ")";
```

Comment:
:+1: Just noting that we're switching from a 24-hour time format to one based on the locale here.  It looks like the time is only used for display in the UI.  (In addition to `ChatMessagePanel`, this text makes its way all the way up to `LobbyGamePanel#getChatLogOfHeadlessHostBot()` via `IModeratorController#getChatLogHeadlessHostBot()` et al.)  I see no code that attempts to parse this text.